content/00.build/30.zksync-sso/40.transaction-fees.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Transaction Fees
+description: Learn which transaction fees apply to ZKsync SSO.
+---
+
+**Account Creation and Recovery Setup:** Free for users.<br>
+The first 1 million users can create accounts and set up recovery options without paying transaction fees, as ZKsync sponsors these fees.
+
+**Session Creation:** When creating application sessions, users pay a transaction fee (roughly equivalent to a regular transaction fee on ZKsync Era).
```

Comment:
"regular transaction fee" is a bit vague, a better wording would be "ERC20 transfer"